### PR TITLE
Issue #4382: Bring back Views localization

### LINKED ISSUE #4382: Views: strings provided via UI aren't translatable at all
**Description of the bug**

In Drupal 7 Views ships with translation (localization) support, but Backdrop has none at all, hence _no_ strings provided via Views UI are translatable.

For instance views labels, titles, header, footer, exposed form elements...

Marking this as a bug, as it's a clear regression compared to D7.

At following point Views came into core, but _all translation handling was stripped_:

https://github.com/backdrop/backdrop-issues/issues/40

The definitions still seem to be there, for example:
https://github.com/backdrop/backdrop/blob/c645dfb16e0a0b054132b70f15ecc13e8e84a63d/core/modules/views/plugins/views_plugin_display.inc#L509

They're useless without the functions that translate (localize) the strings, of course.

views_plugin_localization_core.inc
views_plugin_localization.inc
views_plugin_localization_none.inc

### PR DESCRIPTION
**Pull request by indigoxela**
Fixes https://github.com/backdrop/backdrop-issues/issues/4382

### CODE REVIEW

**Comment by indigoxela** [comment]
Requesting fresh sandbox

**Comment by indigoxela** [comment]
Fresh sandbox, try 2 (deploy timed out)

File: core/modules/views/plugins/views_plugin_localization_core.inc [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+   *   An array of keys to identify the string. Generally constructed from
+   *   view name, display_id, and a property, e.g., 'header'.
+   * @param string $format
+   *   The input format of the string. This is optional.
+   */
+  public function save_string($string, $keys = array(), $format = '') {
+    // Find a language that is not "en" (if available) to trigger an entry
+    // in module locale's db tables, even if current language is "en".
+    $langcode = 'en';
+    $languages = language_list();
```
Comment (herbdool) [inline core/modules/views/plugins/views_plugin_localization_core.inc:45]: Do we want this to only fetch enabled languages? This function changed from D7. If so then you'll need `language_list(TRUE)`.

Comment (indigoxela) [reply core/modules/views/plugins/views_plugin_localization_core.inc:45]: Interesting question. What would be the benefit of only fetching enabled languages?

Locale allows translation even to disabled languages. And D7 Views also fetches all.

Comment (herbdool) [reply core/modules/views/plugins/views_plugin_localization_core.inc:45]: In D7 it fetched all "installed" languages by default https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/language_list/7.x and in Backdrop it fetches all "configured" languages by default https://api.backdropcms.org/api/backdrop/core%21includes%21bootstrap.inc/function/language_list/1 but allows for getting just enabled languages. That's useful in cases like when selecting which language content should be in. I wasn't sure if it was needed here so it's probably fine as is.

**Comment by quicksketch** [comment]
Thanks folks! I've merged this into 1.x and 1.16.x.

**Pull request merged**